soc/nxp/imx/imx8m/Kconfig.soc

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020-2022,2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMX8M
+	bool
+	select SOC_FAMILY_IMX
+
+config SOC_MIMX8MM6
+	bool
+	select SOC_SERIES_IMX8M
+
+config SOC_MIMX8MM6_A53
+	bool
+	select SOC_MIMX8MM6
+	help
+		NXP i.MX8MM A53
+
+config SOC_MIMX8ML8
+	bool
+	select SOC_SERIES_IMX8M
+
+config SOC_MIMX8ML8_A53
+	bool
+	select SOC_MIMX8ML8
+	help
+		NXP i.MX8MP A53
+
+config SOC_MIMX8MN6
+	bool
+	select SOC_SERIES_IMX8M
+
+config SOC_MIMX8MN6_A53
+	bool
+	select SOC_MIMX8MN6
+	help
+		NXP i.MX8MN A53
+
+config SOC_PART_NUMBER_MIMX8ML8DVNLZ
+	bool
+
+config SOC_PART_NUMBER_MIMX8MM6DVTLZ
+	bool
+
+config SOC_PART_NUMBER_MIMX8MM6CVTKZ
+	bool
+
+config SOC_PART_NUMBER_MIMX8MN6DVTJZ
+	bool
+
+config SOC_PART_NUMBER_MIMX8MN6DUCJZ
+	bool
+
+config SOC_PART_NUMBER_MIMX8MN6CVTIZ
+	bool
+
+config SOC_PART_NUMBER_MIMX8MN6CUCIZ
+	bool
+
+config SOC_PART_NUMBER
+	default "MIMX8ML8DVNLZ" if  SOC_PART_NUMBER_MIMX8ML8DVNLZ
+	default "MIMX8MM6DVTLZ" if  SOC_PART_NUMBER_MIMX8MM6DVTLZ
+	default "MIMX8MM6CVTKZ" if  SOC_PART_NUMBER_MIMX8MM6CVTKZ
+	default "MIMX8MN6DVTJZ" if  SOC_PART_NUMBER_MIMX8MN6DVTJZ
+	default "MIMX8MN6DUCJZ" if  SOC_PART_NUMBER_MIMX8MN6DUCJZ
+	default "MIMX8MN6CVTIZ" if  SOC_PART_NUMBER_MIMX8MN6CVTIZ
+	default "MIMX8MN6CUCIZ" if  SOC_PART_NUMBER_MIMX8MN6CUCIZ
+
+config SOC
+	default "mimx8mm6" if SOC_MIMX8MM6
+	default "mimx8mn6" if SOC_MIMX8MN6
+	default "mimx8ml8" if SOC_MIMX8ML8
+
+config SOC_SERIES
+	default "imx8m" if SOC_SERIES_IMX8M
+
+config MCUX_CORE_SUFFIX
+	default "_ca53" if SOC_MIMX8MM6_A53
+	default "_ca53" if SOC_MIMX8MN6_A53
+	default "_ca53" if SOC_MIMX8ML8_A53
```

Comment:
etc.

Reply:
and these can be combined `	default "_ca53" if SOC_MIMX8MM6_A53 || SOC_MIMX8MN6_A53 || SOC_MIMX8ML8_A53`, actually why is this needed at all, why can't you just check `CPU_CORTEX_A53` from your module?

Reply:
Fixed.
Please check https://github.com/zephyrproject-rtos/zephyr/pull/69368 for MCUX_CORE_SUFFIX definition, thanks.